client/src/main/scala/drt/client/components/DeploymentsExportComponent.scala

```diff
@@ -0,0 +1,120 @@
+package drt.client.components
+
+import diode.UseValueEq
+import drt.client.SPAMain
+import drt.client.components.TerminalContentComponent.exportLink
+import drt.client.components.styles.WithScalaCssImplicits
+import drt.client.logger.{Logger, LoggerFactory}
+import drt.client.services._
+import io.kinoplan.scalajs.react.material.ui.core.MuiButton._
+import io.kinoplan.scalajs.react.material.ui.core.{MuiButton, MuiGrid}
+import io.kinoplan.scalajs.react.material.ui.icons.MuiIconsModule.GetApp
+import io.kinoplan.scalajs.react.material.ui.icons._
+import japgolly.scalajs.react.component.Scala.Component
+import japgolly.scalajs.react.vdom.all.onClick.Event
+import japgolly.scalajs.react.vdom.html_<^
+import japgolly.scalajs.react.vdom.html_<^._
+import japgolly.scalajs.react.{Callback, CtorType, ScalaComponent}
+import uk.gov.homeoffice.drt.auth.LoggedInUser
+import uk.gov.homeoffice.drt.auth.Roles.ArrivalsAndSplitsView
+import uk.gov.homeoffice.drt.ports.Terminals.Terminal
+import uk.gov.homeoffice.drt.time.SDateLike
+
+object DeploymentsExportComponent extends WithScalaCssImplicits {
```

Comment:
This class is 99% identical to `ArrivalsExportComponent`. I reckon rather than have so much duplicated code we just amend the `ArrivalsExportComponent` to that it can be used for other exports. It looks like it might only need about 2 additional parameters, and then it can also be used for exporting recommendations too